src/Bling.js

```diff
@@ -547,13 +551,13 @@ class Bling extends Component {
         }
 
         const inViewport = Bling._adManager.isInViewport(
-            ReactDOM.findDOMNode(this),
+            this.wrapper.current,
             slotSize,
             this.viewableThreshold
         );
 
         const isHidden = () => {
-            const el = ReactDOM.findDOMNode(this);
```

Comment:
`findDOMNode` [no longer behaves the way it used to](https://reactjs.org/docs/strict-mode.html#warning-about-deprecated-finddomnode-usage). This looked like the only change needed to get ads rendering. Whether they are rendering _correctly_ I'll leave to you guys ;) at least to tell me that they're broken.